src/openrct2/scenario/Scenario.h

```diff
@@ -278,8 +279,8 @@ struct rct_s6_data
     uint8_t climate;
     uint8_t pad_013CA747;
     uint16_t climate_update_timer;
-    uint8_t current_weather;
-    uint8_t next_weather;
+    WeatherType current_weather;
```

Comment:
Same as the rct1 struct please keep the type as `uint8_t`.

Reply:
Done :)